C/other/Permutation_of_string.c

```diff
@@ -0,0 +1,65 @@
+// program to generate all permutation of the given string
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+//declare permutation and swap functions
+void permutation(char *,int,int);
+void swap(char *,char *);
+int main()
+{
+    char *s;
+    // dynamically creating string length
+    s=(char*)malloc(sizeof(char)*1000);
+    // getting string
```

Comment:
Display a message in print form as what are you taking input from the user for better UI